### ISSUE TITLE
Bot Composer trigger does not work with "User Input (Choice)"

### ISSUE THREAD

**Issue by jrdamiano** [microsoft/BotFramework-Composer#8488]
## Describe the bug

I've realized that triggers does not work when user input is requested via "Ask a question >> Multichoice". If a user writes anything on the bot text box different from "Choice options", trigger does'n work.

## Version

Bot Composer 2.1.0

## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [X] Edge

## OS

- [ ] macOS
- [X] Windows
- [ ] Ubuntu

## To Reproduce

Create a new dialog and create a trigger in Main Dialog to fire the new dialog
In the main dialog, create a "User input (multi-choice)"
Run the bot
While bot is specting for user input choice, write a trigger utterance... and... 
bot repeats input choice until max turn count (if set) is reached, instead of jumping to the new dialog

## Expected behavior

A trigger should fire under any condition, otherwise it is impossible to react to a user input, for instance for a Cancel or Help request

**Comment by jrdamiano** [comment]
Solved...
Reading more docs I've found the way to make this possible by setting "User Input (Choice)" parameter "Allow interruptions" to true. In this way trigger does fire correctly.

**Comment by jrdamiano** [comment]
I've seen another issue when "Allow interruptions" is set to true in "User Input (text)": 
- Global interruptions works fina and, for instance, Cancel or Help triggers works, BUT...
- validation RegEx stops working and it never recognizes user input as a valid input. It always detects "_Interruption" as TopScoring Intent and do not validate user input text, even when it meets Validation Rules

**Comment by benbrown** [comment]
@stevkan were you able to reproduce this?

**Comment by stevkan** [comment]
@jrdamiano, are you using the RegEx recognizer or the default (LUIS) recognizer for your dialogs? Can you post the regex that you are using for your validation so that I can try the same? When testing, the validation is working for me. The only two differences are I am using the default recognizer and for my validation rule I only checked that the user input was a certain length.

**Comment by jrdamiano** [comment]
@stevkan again it's been my begginer's fault... it works fine, really. 

What happened?
At a certain moment, in order to clarify my code, I changed Intent names in User Input LU files BUT I've realized that Intent name CANNOT BE CHANGED and once you create a trigger, that intent name is not modifiable any more... Another thing that affected bot behavior due to Intent name change was the use of Condition parameter as a trigger threshold (#IntentName.score >= 0.7)

So what happened to me is that unspected triggers where fired and other spected ones did not... in summary I created a huge mess and nothing worked fine.

Once I've solved trigger naming, RegEx works fine and validation rules are correctly applied.

Thanks for your time and your kind help.
